.env.test

```diff
@@ -21,6 +21,10 @@ SQS_BRANCH_QUEUE_REGION=us-west-1
 GLOBAL_HASH_SECRET=testsecret
 DEBUG=nock.*
 
+#Dyamno for deployment status
+DYNAMO_DEPLOYMENT_HISTORY_TABLE_REGION=us-west-1
+DYNAMO_DEPLOYMENT_HISTORY_TABLE_NAME=deployment-history-cache
```

Comment:
Should we use "cache" in all places? I see now it is just the name of the table, however "cache" keyword is not propagated in the code....

Reply:
Good catch!! I should have renamed all those now.